java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/client/TaskManagerClient.java

```diff
@@ -57,62 +62,278 @@ public TaskManagerClient(String endpoint) {
         taskManagerInterface = BrpcProxy.getProxy(rpcClient, TaskManagerInterface.class);
         RpcContext.getContext().setLogId(1234L);
     }
-
     /**
      * Stop the brpc client.
      */
     public void stop() {
         rpcClient.stop();
     }
+    /**
+     * Stop the job.
+     *
+     * @param id of job.
+     *
+     * @throws Exception
+     */
+    public String stopJob(int id) throws Exception {
+        TaskManager.StopJobRequest request = TaskManager.StopJobRequest.newBuilder()
+                .setId(id)
+                .build();
+        TaskManager.StopJobResponse response = taskManagerInterface.StopJob(request);
+
+        if (response.getCode() != 0) {
+            String errorMessage = "Fail to request, code: " + response.getCode() + ", error: " + response.getMsg();
+            throw new Exception(errorMessage);
+        }
+        return response.getMsg();
+    }
+    /**
+     * RunBatchSql.
+     *
+     * @param sql,output_path.
```

Comment:
This is not standard Java doc. It would be auto generated by IDE or change like others.

```
@param sql
@param output_path
```